sql/2021/resource-hints/lazy_iframes.sql

```diff
@@ -0,0 +1,39 @@
+#standardSQL
+  # Count the number of lazily loaded iframes
+CREATE TEMPORARY FUNCTION
+countLazyIframes(almanac_string STRING)
+RETURNS INT64
+LANGUAGE js AS ''' 
+try {
+    var almanac = JSON.parse(almanac_string)
+    if (Array.isArray(almanac) || typeof almanac != 'object') return null;
+    
+    var iframes = almanac["iframes"]["iframes"]["nodes"]
+    return iframes.filter(i => (i.loading || "").toLowerCase() === "lazy").length
+}
+catch {
+    return null
+}
+''' ;
+WITH iframe_stats_tb AS (
+  SELECT
+    _TABLE_SUFFIX AS client,
+    JSON_EXTRACT_SCALAR(payload,
+      '$._almanac') AS almanac,
+    countLazyIframes(JSON_EXTRACT_SCALAR(payload,
+        '$._almanac')) AS res
```

Comment:
`almanac` is unused

Also replacing `res` with a more descriptive field name, preferring underscore naming convention for SQL field names

```suggestion
    countLazyIframes(JSON_EXTRACT_SCALAR(payload, '$._almanac')) AS num_lazy_iframes
```